### ISSUE TITLE
DataGrid not rendering when avalonia is running within Autodesk Inventor

### ISSUE THREAD

**Issue by Basnederveen** [AvaloniaCommunity/Material.Avalonia#344]
### Usage Information

Avalonia & Material.Avalonia latest nightly

### Description

When I run my avalonia app regularly, the datagrid is showing. When I start the window from an Autodesk Inventor add-in, it does not. I can see it in the visual inspector though. 
![image](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/16050771/c76c4029-7cfb-4eee-996c-1616fca13bb6)

![image](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/16050771/0b31a62d-a109-4621-bb47-292ac68553bc)

Sample code is just the one from the DataGrid documentation. All other controls seem to be fine. 

I have tried to install all avalonia packages in the 2nd project that is executing the app.

Does anyone have any idea?

### Reproduction Steps

Create a new avalonia app
With material ui and avalonia ui latest nightly builds (had 11.0.6 first, also didn't work)
Run it from another project that is an add-in in a software

### Expected Behavior

Datagrid should render

### Actual Behavior

Datagrid does not render

### Could you help with a pull-request?

I can try

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by SKProCH** [comment]
Hello, @Basnederveen 

I actually can't figure out how Autodesk Invertor and Avalonia connected, please let me know)

About the issue itself: Material.DataGrid styles included actomatically by the MaterialToolkit class (which included by MaterialTheme[Base]):  
![image](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/29896317/4159712b-69b8-4924-8512-16ed5d95cf0b)

So, apparently, there is this flag missing (but it's set automatically when Material.Avalonia.DataGrid presented in project), or the link to the Material.Avalonia.DataGrid itself missing when you launch it inside Invertor.

You can try to explicitly add Material.Avalonia.DataGrid (if it missing now) and add StyleInclude with the following source: `avares://Material.Avalonia.DataGrid/DataGrid.xaml` inside App.Styles in App.axaml.cs.

**Comment by Basnederveen** [comment]
They are not connected. This should apply to any 3rd party software and maybe even any external project - I did not test this, now I am thinking of it.

I made an add-in for inventor where I have some UI's, those I did in Inventor. So I have one regular avalonia project to start the UI's and hooking to inventor but running out of process. With the add-in the avalonia UI runs in the inventor process. 

adding the style in app.xaml worked.. thanks!!
